src/gui/widgets/ControllerRackView.cpp

```diff
@@ -43,32 +44,32 @@
 
 
 ControllerRackView::ControllerRackView( ) :
-	QWidget()
+	QWidget(),
+	m_nextIndex(0)
 {
-	setMinimumWidth( 250 );
-	setMaximumWidth( 250 );
-	resize( 250, 160 );
-
 	setWindowIcon( embed::getIconPixmap( "controller" ) );
 	setWindowTitle( tr( "Controller Rack" ) );
 
 	m_scrollArea = new QScrollArea( this );
-	m_scrollArea->setVerticalScrollBarPolicy( Qt::ScrollBarAlwaysOn );
-	m_scrollArea->setHorizontalScrollBarPolicy( Qt::ScrollBarAlwaysOff );
```

Comment:
I'm not sure I agree with this.

Previously, it was impossible to resize the controller rack small enough to where it had a horizontal scrollbar. Now it is possible. I prefer the old behavior, mainly because now when I open an empty controller rack & add a new controller, the window will not be large enough to fit all of it.

I'm don't know if this is a result of removing the setHorizontalScrollBarPolicy or setMinimumWidth calls. FWIW I do agree with you making the vertical scrollbar behavior automatic.